newsfragments/3584.bugfix

```diff
@@ -0,0 +1 @@
+SFTP public key auth likely works better, and SFTP in general was broken in the prerelease.
```

Comment:
I'm not sure these changes will make it into 1.16 final, so it might not be clear which prerelease this refers to.  Maybe just say "was broken".